index.js

```diff
@@ -50,17 +51,22 @@ const lookupHandler = function (params) {
             return;
         }
 
+        let userAlreadyInQueue = queue.isUserInQueue(params.ip);
+
         // If the flag is set, check if the user already has a request in the queue
         if (!CONFIG.allow_simultaneous_requests && userAlreadyInQueue) {
```

Comment:
Since we don't use `userAlreadyInQueue` anywhere else, I think we could just replace `userAlreadyInQueue` with `queue.isUserInQueue(params.ip)` here.

It is just some legacy stuff from when we had to check the status of the user in the queue before adding them in `res_controller`.